src/vl-map.js

```diff
@@ -58,12 +58,25 @@ export class VlMap extends vlElement(HTMLElement) {
   /**
    * Geeft de OpenLayers map terug.
    *
-   * @Return {VlCustomMap}
+   * @return {VlCustomMap}
    */
   get map() {
     return this._map;
   }
 
+  /**
+   * Geeft de OpenLayers kaart visualisatie terug.
+   *
+   * @return {Object}
+   */
+  get view() {
+    return this._map.getView();
+  }
```

Comment:
misschien ook niet nodig om deze te exposen voorlopig? ik denk alleen maar getResolution() nodig for now? 
ik zou zo weinig mogelijk (als praktisch haalbaar) proberen directly te exposen van de openlayers stuff.

Reply:
Prima, ik pas het aan.

Reply:
in praktijk gaat ge wschl uiteindelijk toch uitkomen met dat ge de openlayers map gaat exposen op map, en de openlayers layer gaat exposen op layer, omdat afnemers toch ergens ne keer iets exotisch willen doen, maar ik zou het beperken tot het niet meer kan :-)